js-exercises/.eslintrc

```diff
@@ -1,12 +0,0 @@
-{
```

Comment:
js-exercises are not part of current branch.
They are kept to be part of master branch, so that any new js-exercise started, can take help of these files. 😋